QuickArterySegmentation/QuickArterySegmentation.py

```diff
@@ -119,7 +118,9 @@ def setup(self):
     shortcut.setKey(qt.QKeySequence('Meta+d'))
     shortcut.connect( 'activated()', lambda: self.removeOutputNodes())
     
-    self.installExtensionFromServer("SegmentEditorExtraEffects")
+    # Avoid cdash test failure.
+    if slicer.app.layoutManager():
+        self.installExtensionFromServer("SegmentEditorExtraEffects")
```

Comment:
This call can raise exceptions. Catch and process them (e.g., you can log it and/or display it to the user - of course pay attention not to show any popup in testing mode) in this method, expecially because this is a callback method from C++.